kong.conf.default

```diff
@@ -48,6 +48,13 @@
 # NGINX
 #------------------------------------------------------------------------------
 
+#set_real_ip_from = 0.0.0.0/0;   # Defines trusted addresses that are known to send
+				 # correct replacement addresses. 
+				 # If the special value unix: is specified, 
+				 # all UNIX-domain sockets will be trusted.
+				 # See: http://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from
```

Comment:
Also, please include # See [link] on the next line, like so: https://github.com/Mashape/kong/blob/master/kong.conf.default#L34-L35

```
# Note: See http://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from
# for a list of accepted values.
```